src/_product-updates/2024-06-15-love-cycle.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Love Cycle - June 24
+tags: [Improvement]
+image: /images/changelog/love-cycle-june-2024.png
+---
+
+![love-cycle.png](/images/changelog/love-cycle-june-2024.png)
+
+The last few weeks, we have been busy improving Bump.sh through small enhancements, specific user requests, while globally polishing the product. We call these moments "Love Cycles".
+
+Here is the list of every small change we made.
+
+As always, we love to hear about any features you would like to see added to the product. Send us a message at hello (at) bump.sh!
+
+Improvements: 
+- When you are subscribed to an API changelog, the link inside the email notifying about a new change now redirects to the specific change page. 
+- We improved the display of oneOf/anyOf alternatives. When there is a lot of alternatives, we now display the list inside a select instead of a horizontally scrolling tab list.
+- The API/Hub visibility has been reworked, to mitigate the risk of mistakes. The visibility of a documentation inside a private hub is now locked to private.
+- Single change: we worked on the page to remove duplicated information, and put the focus the deployment date and change id, that can both be useful depending on who reads it. We also put some love on the mobile version.
```

Comment:
I am still confused about talking about "single change". It might not be the most appropriate vocabulary. 

I don't mean to change everything now, but it seems more relevant to talk about a "changelog entry" which clearly refers to an item in a list of multiple items, rather than a "single change" which would apply to a different context as "something that would happen only once".

Reply:
That could be better, yes! 
I agree that it's confusing, I left a message talking about it on the PR of the Product Update. (cc @jayk-u )

Reply:
What about “Single changelog entry”, Or “Changelog event”?

Reply:
Do we need the "single"? It feels like as we say "Changelog" for the global listing, a "Changelog entry" logically becomes an item of this global listing? Wdyt? 

There's a lot of resources talking about a "Changelog entry"
https://keepachangelog.com/en/1.1.0/
https://docs.gitlab.com/ee/development/changelog.html

Reply:
Do we still want to keep this part in the Love Cycle PU, as it has its own entry in the changelog?

Reply:
Hey! It's gonna be an enhancement of what we shipped a few weeks ago